libr/util/unum.c

```diff
@@ -853,3 +851,8 @@ R_API void *r_num_dup(ut64 n) {
 	*hn = n;
 	return (void*)hn;
 }
+
+R_API st64 r_num_abs(st64 num) {
```

Comment:
why not put this function. in the .h as static inline? i cant think of any. reason to have this primitive not inlined, and shouldn't be available to the bindings. imho

Reply:
agreed